website/addons/osfstorage/tests/test_models.py

```diff
@@ -15,61 +14,61 @@
 from modularodm import exceptions as modm_errors
 
 
+from website.files import models
 from website.addons.osfstorage import utils
-from website.addons.osfstorage import model
 from website.addons.osfstorage import settings
 
 
-class TestFileGuid(OsfTestCase):
-    def setUp(self):
-        super(OsfTestCase, self).setUp()
-        self.user = factories.AuthUserFactory()
-        self.project = ProjectFactory(creator=self.user)
-        self.node_addon = self.project.get_addon('osfstorage')
-
-    def test_provider(self):
-        assert_equal('osfstorage', model.OsfStorageGuidFile().provider)
-
-    def test_correct_path(self):
-        guid = model.OsfStorageGuidFile(node=self.project, path='/baz/foo/bar')
-
-        assert_equals(guid.path, '/baz/foo/bar')
-        assert_equal(guid.path, guid.waterbutler_path)
-        assert_equals(guid.waterbutler_path, '/baz/foo/bar')
-
-    @mock.patch('website.addons.base.requests.get')
-    def test_unique_identifier(self, mock_get):
-        mock_response = mock.Mock(ok=True, status_code=200)
-        mock_get.return_value = mock_response
-        mock_response.json.return_value = {
-            'data': {
-                'name': 'Morty',
-                'extra': {
-                    'version': 'Terran it up'
-                }
-            }
-        }
+# class TestFileGuid(OsfTestCase):
```

Comment:
Lots of commented-out tests in this file - as I suggested above, let's either go ahead and remove them, or comment as to why they're commented out.